src/shared/tpm2-util.c

```diff
@@ -12,8 +12,10 @@
 #include "format-table.h"
 #include "fs-util.h"
 #include "hexdecoct.h"
+#include "hmac.h"
```

Comment:
this is not actually used, is it? i.e. you opted not to do HMAC() of the passed password after all, right?

Reply:
It's only used for the `SHA256_DIGEST_SIZE` define. I wonder why this isn't in sha256.h. Maybe it should be moved there?

Reply:
move it over, please!